config/firstalert/zcombo-g.xml

```diff
@@ -0,0 +1,41 @@
+<Product Revision="3"
+  xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="Name">Battery Smoke/CO Combo Alarm</MetaDataItem>
+    <MetaDataItem name="Description">ZWave enabled, battery-powered combination smoke and carbon monoxide alarm</MetaDataItem>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0138:0003:0001</MetaDataItem>
+    <MetaDataItem name="ProductPage">https://www.firstalertstore.com/store/products/z-wave-smoke-and-carbon-monoxide-alarm-zcombo-g.htm</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/firstalert/zcombo.png</MetaDataItem>
+    <MetaDataItem name="ProductManual">https://www.firstalertstore.com/store/images/pdf/zcombo-g-first-alert-z-wave-smoke-and-co-alarm-manual.pdf</MetaDataItem>
+    <MetaDataItem name="WakeupDescription">To manual wake-up the device, slide the battery door open wait for ~5 seconds and then slide the battery drawer closed. Upon power-up, the device will send the wake-up notification.</MetaDataItem>
+    <MetaDataItem name="InclusionDescription">1. Slide battery door open.
+2. Insert batteries checking the correct orientation.
+3. Press and hold the test button. Keep it held down as you slide the battery drawer closed. You may then release the button.</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">1. Slide battery door open.
+2. Insert batteries checking the correct orientation.
+3. Press and hold the test button. Keep it held down as you slide the battery drawer closed. You may then release the button.</MetaDataItem>
+    <MetaDataItem name="ResetDescription">If the device is powered up with the test button held down for 10+ seconds, the device will reset all Z-Wave settings and leave the network.
+Upon completion of the Reset operation, the LED will glow and the horn will sound for ~1 second.
+Please use this procedure only when the network primary controller is missing or otherwise inoperable.
+    </MetaDataItem>
+    <MetaDataItem id="0003" name="ZWProductPage" type="0001">https://products.z-wavealliance.org/products/3886/</MetaDataItem>
+    <MetaDataItem id="0003" name="FrequencyName" type="0001">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0003" name="Identifier" type="0001">ZCOMBO-G</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Richard Tallent - richard@tallent.us" date="10 Oct 2020" revision="1">Copy from ZCOMBO, update based on Z-Wave Alliance Database - https://products.z-wavealliance.org/products/3886/xml</Entry>
+      <Entry author="Richard Tallent - richard@tallent.us" date="10 Oct 2020" revision="2">Fix XML</Entry>
+    </ChangeLog>
+  </MetaData>
+  <CommandClass id="112">
+    <Value genre="config" index="1" instance="1" label="Send double alarm messages" size="1" type="list" value="0">
```

Comment:
I did copy the CommandClass info from the previous FirstAlert device, but it seemed (to my ZWave newbie eye) to match the info on the [manufacturer's product page](https://www.firstalertstore.com/store/products/z-wave-smoke-and-carbon-monoxide-alarm-zcombo-g.htm) under the "Z-Wave Setup" tab:

```
The alarm supports one association group with up to 1 node, and sends its alarms to that group.
The alarm supports configuration parameter #1, which has a size of 1, a default value of 0, and when set to 1 causes the device to send double alarm messages.
```

Sounds like the info you linked would be more authoritative, I guess they just did some copy/paste on the marketing site? I think I can sort my way through getting those added.

Reply:
Not that it's 100% perfect, but I would trust the zwave DB and the spec document over the web store. The web store looks like it's a third party that just licenses the First Alert name. Strangely, these Z-Wave alarms don't even show up at firstalert.com, but if you search for Z-wave you get the spec doc...

Otherwise, you'd have to test out the parameters yourself and see if they are accepted or not. I'd be happy to test when I get mine (assuming they are actually the new one).